### DISCUSSION TITLE
Receiving e-mail by HTTP getting only base64 string

### DISCUSSION THREAD

**Discussion by Muhammedalihudaybergenow** [postalserver/postal#2498] [Help with using Postal]
how to get the processed payload, i only get base 64 string
![Снимок экрана (4)](https://github.com/postalserver/postal/assets/94282427/388b014e-c602-46a9-a625-6bc544f4ecb5)
when i configure myroute this i get message like base64 string i wand to get json processed payload as written in documentation how can i get like that

**Comment by willpower232** [comment]
I think you need to change this setting to hash

![image](https://github.com/postalserver/postal/assets/1619102/7aad7917-2d3b-466a-a819-da77007d49f5)

**Comment by Muhammedalihudaybergenow** [comment]
Thanx it worked for me
